src/js/webrequest.js

```diff
@@ -465,16 +448,15 @@ function recordFingerprinting(tabId, msg) {
 /**
  * Read the frame data from memory
  *
- * @param tabId TabId to check for
- * @param frameId FrameID to check for
+ * @param tab_id Tab ID to check for
```

Comment:
I prefer `lowercase_and_underscores` over `mixedCase` names *so much*, but it is inconsistent with the rest of the code.